stdlib/2and3/argparse.pyi

```diff
@@ -117,6 +117,7 @@ class Action:
                  option_string: _Text = ...) -> None: ...
 
 class Namespace:
+    def __init__(self, **kwargs: Dict[_Text, Any]) -> None: ...
```

Comment:
This means that every keyword argument must be a Dict. I think you want just `**kwargs: Any`. (That would also fix the Travis failures.)